README.md

```diff
@@ -172,11 +172,11 @@ The function returns the data for the template.
 
 `Function`
 
-The function allows to extract step datas to save in the browser storage and persist during the navigation.
+The function allows to extract step data to save in the browser storage and persist during the navigation.
```

Comment:
_...browser storage in order to persist during the entire navigation._

Reply:
C'est corrigé